packages/windows/data_stream/applocker_exe_and_dll/elasticsearch/ingest_pipeline/default.yml

```
@@ -220,6 +143,29 @@ processors:
             field: winlog.user_data.TargetProcessId
             ignore_failure: true
 
+    ## Fully Qualified Binary Name (FQBN) / File Fields
+  - grok:
+      field: winlog.user_data.Fqbn
+      ignore_missing: true
+      patterns:
+      - "O=(?<file.x509.subject.organization>.*),%{SPACE}L=(?<file.x509.subject.locality>.*),%{SPACE}S=(?<file.x509.subject.state_or_province>.*),%{SPACE}C=(?<file.x509.subject.country>[^\\\\]*)\\\\(?<file.pe.product>[^\\\\]*)\\\\(?<file.pe.original_file_name>[^\\\\]*)\\\\(?<file.pe.file_version>.*)"
```

Can you please anchors these patterns (reasoning is in https://www.elastic.co/blog/do-you-grok-grok). For example:

```suggestion
      - "^O=(?<file.x509.subject.organization>.*),%{SPACE}L=(?<file.x509.subject.locality>.*),%{SPACE}S=(?<file.x509.subject.state_or_province>.*),%{SPACE}C=(?<file.x509.subject.country>[^\\\\]*)\\\\(?<file.pe.product>[^\\\\]*)\\\\(?<file.pe.original_file_name>[^\\\\]*)\\\\(?<file.pe.file_version>.*)"
```

---

Added